src/main/java/core/basesyntax/dao/ConverterDataToObjImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.FruitTransaction;
+import java.util.List;
+
+public class ConverterDataToObjImpl {
+    private static final String SPLITERATOR = ",";
```

Comment:
"," is a separator, that has no common with Spliterator:
```suggestion
    private static final String SEPARATOR = ",";
```